@xen-orchestra/web-core/lib/components/ui/top-bottom-table/UiTopBottomTable.vue

```diff
@@ -0,0 +1,52 @@
+<!-- v2 -->
+<template>
+  <div class="ui-top-bottom-table">
+    <div class="content">
+      <span class="typo p3-regular label">
+        {{ $t('core.select.n-selected', { count: selectedItems, total: totalItems }) }}
+      </span>
+
+      <UiButton accent="info" size="small" variant="tertiary" @click="emit('toggleSelectAll', true)">
+        {{ $t('core.select.all') }}
+      </UiButton>
+      <UiButton accent="info" size="small" variant="tertiary" @click="emit('toggleSelectAll', false)">
+        {{ $t('core.select.unselect') }}
+      </UiButton>
```

Comment:
@clemencebx What would you think of displaying the “Select all” button when `selectedItems < totalItems` and “Unselect all” when `selectedItems > 0`?

I think it's strange to keep "select all" when all items are already selected or "unselect all" when none are selected.

```suggestion
      <UiButton v-if="selectedItems < totalItems" accent="info" size="small" variant="tertiary" @click="emit('toggleSelectAll', true)">
        {{ $t('core.select.all') }}
      </UiButton>
      <UiButton v-if="selectedItems > 0" accent="info" size="small" variant="tertiary" @click="emit('toggleSelectAll', false)">
        {{ $t('core.select.unselect') }}
      </UiButton>
```

Reply:
Is it possible to change the buttons to .disabled in those cases so we keep the buttons but not the interactions?

Reply:
Sure

Reply:
There’s a general issue with the text color for disabled interactions. We’ll correct this directly in the Design System, which will automatically apply the fix here as well, but the behavior is what we are looking for :)